### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

Spelling fixes

You can test using `fchurn`, but it still has to be manual...

### CODE REVIEW

**Comment by ChrisLundquist** [comment]
```
./builder/alicloud/ecs/step_config_public_ip.go
You can use the command: `make fmt` to reformat code.
```
Looks like the build failed on formatting
LXD changes look good.  Thanks!

File: builder/alicloud/ecs/step_config_public_ip.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 )
 
 type stepConfigAlicloudPublicIP struct {
-	publicIPAdress string
+	publicIPAddress string
```
Comment (ChrisLundquist) [inline builder/alicloud/ecs/step_config_public_ip.go]: yeah formatting broke here I think since the type isn't aligned.

**Comment by jsoref** [comment]
@ChrisLundquist: updated

File: builder/virtualbox/common/export_config.go [hunk 2]
```diff
@@ -7,7 +7,7 @@ import (
 )
 
 type ExportConfig struct {
-	Format string `mapstruture:"format"`
+	Format string `mapstructure:"format"`
```
Comment (SwampDragons) [inline builder/virtualbox/common/export_config.go]: Oh wow. Shocked we haven't seen more bugs from this.

File: builder/hyperv/common/step_polling_installation.go [hunk 3]
```diff
@@ -15,10 +15,10 @@ import (
 
 const port string = "13000"
 
-type StepPollingInstalation struct {
+type StepPollingInstallation struct {
```
Comment (SwampDragons) [inline builder/hyperv/common/step_polling_installation.go]: Come to think of it, it looks like this file isn't even in use in Packer anymore.  @taliesins should I just delete the file? StepPollingInstalation isn't used.

**Comment by SwampDragons** [comment]
Thanks for the spring cleaning!

**Pull request merged**